File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ArtInstituteOfChicago SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ArtInstituteOfChicagoMakeContext
{
    public static function call(array $ctxmap, ?ArtInstituteOfChicagoContext $basectx): ArtInstituteOfChicagoContext
    {
        return new ArtInstituteOfChicagoContext($ctxmap, $basectx);
    }
}

File: py/artinstituteofchicago_sdk/utility/make_context.py
# ArtInstituteOfChicago SDK utility: make_context

from artinstituteofchicago_sdk.core.context import ArtInstituteOfChicagoContext


def make_context_util(ctxmap, basectx):
    return ArtInstituteOfChicagoContext(ctxmap, basectx)
